src/api/repos/stargazers.rs

```diff
@@ -32,12 +32,16 @@ impl<'octo, 'r> ListStarGazersBuilder<'octo, 'r> {
   }
 
   /// Sends the actual request.
-  pub async fn send(self) -> crate::Result<crate::Page<crate::models::User>> {
+  pub async fn send(self) -> crate::Result<crate::Page<crate::models::StarGazer>> {
     let url = format!(
       "repos/{owner}/{repo}/stargazers",
       owner = self.handler.owner,
       repo = self.handler.repo
     );
-    self.handler.crab.get(url, Some(&self)).await
+
+    let mut headers = reqwest::header::HeaderMap::new();
+    headers.insert(ACCEPT, "application/vnd.github.v3.star+json".parse().unwrap());
```

Comment:
```suggestion
    headers.insert(ACCEPT, "application/vnd.github.star+json".parse().unwrap());
```

nit (feel free to ignore): While it does not affect behaviour, the docs recommend to not use `v3`. See https://docs.github.com/en/rest/activity/starring#custom-media-types-for-starring and https://docs.github.com/en/rest/overview/media-types ("Note: In the past, we recommended including v3 in your Accept header. This is no longer required and will have no impact on your API requests.")